src/jsonValidator.js

```diff
@@ -0,0 +1,12 @@
+import { createError } from './error.js';
+
+export const jsonIsValid = (jsonString, errors) => {
+  const valid= true;
+  try {
+    JSON.parse(jsonString);
+  } catch (error) {
+    errors.push(createError("invalid-json"));
+    return !valid;
```

Comment:
If we are using a variable, we should update the variable value here instead of doing a `return`.

Reply:
@mpollingerQualabs . Sorrry but we don´t understand the idea. Could you explain it better?